terraform/modules/linode/k8s/main.tf

```
@@ -66,3 +67,17 @@ resource "kubernetes_secret" "tls" {
     "tls.key" = var.tls_keys[each.key],
   }
 }
+# S3 CSI credentials
+resource "kubernetes_secret" "s3_csi" {
+  metadata {
+    name      = "csi-s3-secret"
+    namespace = "kube-system"
+  }
+
+  data = {
+    "endpoint"        = var.s3_csi.s3_endpoint,
+    "accessKeyID"     = var.s3_csi.access_key_id,
+    "secretAccessKey" = var.s3_csi.access_key,
+    "region"          = ""
```

```suggestion
    # as we are not using Amazon S3, the region field is not used
    "region"          = ""
```
Document why the `region` field is not used.